docs/source/pillars/computing_technology.md

```diff
@@ -2,55 +2,55 @@
 
 # Computing technology
 
-This capability concerns what the TRE operator does to manage systems for storing, retrieving, analysing and sending information.
-
 ```{figure} ../../images/Capability_Map/full.drawio.svg
 :alt: SATRE Pillars Capability Map
 :align: center
 
 SATRE Pillars Capability Map
 ```
 
+This pillar concerns actions taken by the TRE operator to manage systems for storing, retrieving, analysing and sending information.
+
+Each TRE operator will have its own computing technology requirements.
+Requirements are drawn from information governance requirements which primarily specify the controls needed within the technical and computing infrastructure.
+Researcher personas influence the according to the technical knowledge and experience of researchers along with the work they need to perform within the system, a data scientist will have very different requirements to a clinician for example.
```

Comment:
I think this paragraph might be aiming for something like this?

```suggestion
Each TRE operator will have its own computing technology requirements.
Information governance requirements primarily specify security controls needed within the technical and computing infrastructure.
Other computing technology requirements will be influenced by the technical knowledge and experience of those using the TRE, along with the work they need to perform within the system. 
For example, a data scientist will have very different requirements to a clinician.
```

Reply:
Are you happy with this @JimMadge ?

Reply:
Sounds good to me :+1:.

Might just be tempted to remove some of the adjectives for brevity.